documentation/modules/proc-configuring-mirror-maker.adoc

```diff
@@ -114,8 +114,9 @@ spec:
                       - postgresql
                       - mongodb
               topologyKey: "kubernetes.io/hostname"
+  tracing: <20>
+    type: jaeger
```

Comment:
if adding Jaeger to MirrorMaker configuration, then you should add the env vars as well. Am I right @scholzj ?

Reply:
I guess so. I've added.